common/src/main/scala/explore/components/primereact/FormInputText.scala

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components.primereact
+
+import cats.syntax.all.*
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.TopNode
+import japgolly.scalajs.react.vdom.html_<^.*
+import org.scalajs.dom.Element
+import org.scalajs.dom.HTMLInputElement
+import react.common.*
+import react.floatingui
+import react.floatingui.syntax.*
+import react.primereact.InputText
+import react.primereact.PrimeStyles
+import reactST.primereact.components.{Button => CButton}
+
+import scalajs.js
+
+final case class FormInputText(
+  id:               NonEmptyString,
+  value:            js.UndefOr[String] = js.undefined,
+  label:            js.UndefOr[TagMod] = js.undefined,
```

Comment:
Having the label here can reduce some code while using, but more important in my opinion is that it ensures that the id of the input matches the `htmlFor` of the label for accessibility reasons.